examples/webrx/index.html

```diff
@@ -0,0 +1,79 @@
+</head>
+
+<body>
+    <section id="todoapp">
+        <header id="header">
+            <h1>todos</h1>
+            <input autofocus="" data-bind="textInput: @current, 
+                keyPress: { 'enter': { command: addCmd, parameter: $data } }"
+                id="new-todo" placeholder="What needs to be done?">
+        </header>
+
+        <section data-bind="visible: todos.length" id="main">
+            <input data-bind="checked: remainingCount === 0, command: completeAllCmd" id="toggle-all" type="checkbox"> 
+            <label for="toggle-all">Mark all as complete</label>
+
+            <ul data-bind="foreach: filteredTodos" id="todo-list">
+                <li data-bind="css: { completed: completed, editing: editing }">
+                    <div class="view">
+                        <input class="toggle" data-bind="checked: @completed" type="checkbox">
+                        <label data-bind="text: title, event: { 
+                            dblclick: { command: $root.editItemCmd, parameter: $data } }">
+                        </label> 
+                        <button class="destroy" data-bind="command: { command: $root.removeCmd, parameter: $data }">
+                        </button>
+                    </div>
+                    
+                    <input class="edit" data-bind="textInput: @title, 
+                        hasFocus: { property: editing, delay: true }, 
+                        event: { 
+                            blur: { command: $root.saveEditingCmd, parameter: $data } }, 
+                        keyPress: { 
+                            'enter': { command: $root.saveEditingCmd, parameter: $data }, 
+                            'esc': { command: $root.cancelEditingCmd, parameter: $data } }">
+                </li>
+            </ul>
+        </section>
+
+        <footer data-bind="visible: completedCount || remainingCount" id="footer">
+            <span id="todo-count"><strong data-bind="text: remainingCount">0</strong> 
+            <span data-bind="text: remainingCount === 1 ? 'item' : 'items'"></span> left</span>
+
+            <ul id="filters">
+                <li>
+                    <a data-bind="sactive: { name: '$', cssClass: 'selected' }, sref: '$'">All</a>
+                </li>
+
+                <li>
+                    <a data-bind="sactive: { name: 'active', cssClass: 'selected' }, sref: 'active'">Active</a>
+                </li>
+
+                <li>
+                    <a data-bind="sactive: { name: 'completed', cssClass: 'selected' }, sref: 'completed'">Completed</a>
+                </li>
+            </ul>
+            
+            <button data-bind="visible: completedCount, command: removeCompletedCmd" id="clear-completed">Clear completed</button>
+        </footer>
+    </section>
+
+    <footer id="info">
+        <p>Double-click to edit a todo</p>
+        <p>Written by <a href="http://webrxjs.org">Oliver Weichhold</a></p>
```

Comment:
this needs to link to a personal website/github profile or whatever you prefer. Not the framework itself.